lib/pengine/rules.c

```diff
@@ -1044,7 +1071,8 @@ compare_attr_expr_vals(const char *l_val, const char *r_val, const char *type,
  * \param[in]   l_val   Value on left-hand side of comparison
  * \param[in]   r_val   Value on right-hand side of comparison
  * \param[in]   type    How to interpret the values (allowed values:
- *                      \c "string", \c "number", \c "version", \c NULL)
+ *                      \c "string", \c "number", \c "version",
+ *                      \c "version", \c NULL)
```

Comment:
I think you meant "integer" for one of these :)

Reply:
Sure did.